compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/expressions/BLangQueryExpr.java

```diff
@@ -46,6 +46,8 @@ public class BLangQueryExpr extends BLangExpression implements QueryExpressionNo
     public boolean isTable = false;
     public boolean isMap = false;
 
+    public BLangStatementExpression stmtExpr;
```

Comment:
Is this change currently unused?

Reply:
https://github.com/ballerina-platform/ballerina-lang/pull/37137/files#diff-a5961a28254babde0aa44e4c1cb05126b5ee68494baccca1c76879dec267b8aeR8092 it is used here